### DISCUSSION TITLE
[Vision] Business Model

### DISCUSSION THREAD

**Discussion by brillout** [concrete-utopia/utopia#1403] [General]
Curious about your vision on this.

A straightforward way would be to monetize deployment as non-programmers are not comfortable setting up a static host such as Netlify.

But I can see more effective ways to monetize.

I really like your stance of sticking to the MIT License and keeping everything free for "the small guy".

**Comment by localhosted** [comment]
I am thinking of setting this up for our semi-technical founder, we are currently paying too much for renderforest, it'd be nice to support an open source project while having a similar website builder/hosting

This is great for building landing pages without having to involve developers. I'd only expect there to be a large collection of design templates & components
